echidna/scripts/zero_nu_limit.py

```diff
@@ -85,18 +84,49 @@ def __call__(self, parser, namespace, values, option_string=None):
     shrunk = Te130_0n2b.sum()
     scaling = shrunk/unshrunk
 
-    Te130_0n2b = store.load(args.signal)
+    # Set decay converter
+    atm_weight_iso = 129.9062244
+    atm_weight_nat = 127.603
+    abundance = 0.3408
+
+    phase_space = 3.69e-14  # Kotila & Iachello 2012
+    matrix_element = 4.03
+
+    converter = decay.DBIsotope(
+        "Te130", atm_weight_iso, atm_weight_nat, abundance, phase_space,
+        matrix_element, roi_efficiency=scaling)
+
+    Te130_0n2b = store.load(args.signal)  # Reload spectrum
 
     # Create background spectra
     Te130_2n2b = store.load(args.two_nu)
-    B8_Solar = store.load(args.b8_solar)
+    Te130_2n2b.scale(1.e6)
+    unshrunk = Te130_2n2b.sum()
+    Te130_2n2b = store.load(args.two_nu)
+    Te130_2n2b.scale(1.e6)
+    shrink_dict = {"energy_mc_low": roi[0], "energy_mc_high": roi[1]}
+    Te130_2n2b.shrink(**shrink_dict)
+    shrunk = Te130_2n2b.sum()
+    scaling = shrunk / unshrunk
 
+    # Set decay converter
+    phase_space = 1.529e-18  # Kotila & Iachello 2012
+    matrix_element = 3.31  # Barea et al. 2013
+
+    two_nu_converter = decay.DBIsotope(
+        "Te130_2n2b", atm_weight_iso, atm_weight_nat, abundance, phase_space,
+        matrix_element, roi_efficiency=scaling)
+
+    Te130_2n2b = store.load(args.two_nu)  # Reload spectra
+    B8_Solar = store.load(args.b8_solar)
 
     # 1/ Set limit with no penalty term
     # Create dictionary of backgrounds and priors
-    Te130_2n2b_prior = 37.396e6  # Based on NEMO-3 T_1/2, for 10 years
-    # from integrating whole spectrum scaled to Valentina's number
-    B8_Solar_prior = 12529.9691
+    two_nu_half_life = 6.9e20  # NEMO-3
```

Comment:
Minor difference but which paper did you get this from? This one has it at 7.0e20 http://discovery.ucl.ac.uk/1304503/1/1304503.pdf